### PR TITLE
New run script; python3; restructure

### PR DESCRIPTION
**Pull request by nottrobin**
Many changes:

- Add run script
- Support python 3
- Use an encapsulated swift service, so you don't need your own swift account
- Improve and convert tests to django tests
- Improve test scripts
- Update README

QA
--

``` bash
./run
./run test
./run clean
etc.
```

### CODE REVIEW

**Comment by nottrobin** [comment]
[Add ./run script](https://trello.com/c/FWgG3qA2/3343-add-run-script)

File: README.md [hunk 1]
```diff
@@ -89,15 +56,15 @@ You can also specify multiple comma separated operations. They will be applied i
 To interact with the assets server, you'll need to generate an authentication token:
 
 ``` bash
-$ scripts/get-token.sh your-name  # Generate a new token called "your-name"
-Token 'your-name' created
+$ ./run django manage.py gettoken {token-name}
```
Comment (WillMoggridge) [inline README.md]: I had to do `./run django python manage.py gettoken {token-name}`

Comment (nottrobin) [inline README.md]: Ah yes. Or `./run django ./manage.py gettoken`

**Comment by nottrobin** [comment]
@WillMoggridge please have another look when you get a chance.

**Pull request merged**